### PR TITLE
Few updates to FRB build

### PR DESCRIPTION
**Pull request by keheintz**
As titled.

### CODE REVIEW

File: frb/builds/build_frbs.py [hunk 1]
```diff
@@ -21,10 +21,18 @@ def frb_121102():
                         z_frb=0.19273)
     # NE2001
     frb121102.set_DMISM()
+
+    # FRB properties
+    frb121102.Fluence = 0.1 * units.Jy * units.ms    # Lower limit from Chatterjee+17/Bhandari+20
+    frb121102.Fluence_err = -999. * units.Jy * units.ms # -- //-- 
+    frb121102.RM = 1.e5 * units.rad / units.m**2
+    frb121102.width = 3.0 * units.ms
```
Comment (profxj) [inline frb/builds/build_frbs.py:29]: There is a `set_width()` method.  Use that.
And modify it as you need.

File: frb/builds/build_frbs.py [hunk 2]
```diff
@@ -48,10 +56,15 @@ def frb_180916():
...
-    #frb180924.fluence_err = 1 * units.Jy * units.ms
-    #
+    
+    # FRB properties
+    frb180916.Fluence = 2.53 * units.Jy * units.ms    
+    frb180916.RM = -114.6 * units.rad / units.m**2
+    frb180916.RM_err = 0.6 * units.rad / units.m**2
+    frb180916.width = 1.66 * units.ms
+    frb180916.width_err = 0.05 * units.ms
+    frb180916.tau = 0.003 * units.ms
```
Comment (profxj) [inline frb/builds/build_frbs.py:66]: What is tau?

File: frb/builds/build_frbs.py [hunk 3]
```diff
@@ -21,10 +21,18 @@ def frb_121102():
                         z_frb=0.19273)
     # NE2001
     frb121102.set_DMISM()
+
+    # FRB properties
+    frb121102.Fluence = 0.1 * units.Jy * units.ms    # Lower limit from Chatterjee+17/Bhandari+20
```
Comment (profxj) [inline frb/builds/build_frbs.py:26]: Fluence is lower case in the code.  Be sure to
look at `GenericFRB` class in `frb.py`

**Review by profxj — CHANGES_REQUESTED** [review]
A few comments for you.

File: frb/builds/build_frbs.py (hunk 2 above)
Comment (keheintz) [reply frb/builds/build_frbs.py:66]: The scattering time scale.

**Pull request closed without merging**